src/test/configuration_snippet.rs

```diff
@@ -96,6 +96,9 @@ impl ConfigCodeBlock {
 
     fn get_block_config(&self) -> Config {
         let mut config = Config::default();
+        if !crate::is_nightly_channel!() {
+            config.override_value("unstable_features", "true");
+        }
```

Comment:
Figured this would be better than just disabling the cfg snippet tests outright on beta